packages/plugin-relay/src/schema-builder.ts

```diff
@@ -156,6 +156,7 @@ schemaBuilderProto.nodeInterfaceRef = function nodeInterfaceRef() {
           id: t.arg.id({ required: true }),
         },
         resolve: async (root, args, context, info) =>
+          this.options.relayOptions?.nodeQueryOptions?.resolve?.(String(args.id), context) ??
```

Comment:
this will fallback to the default resolver if the custom resolver returns null or undefined